code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -151,6 +153,48 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 200, 55)
 
+/obj/item/reagent_containers/food/drinks/bottle/imitation
+	name = "imitation glass bottle"
```

Comment:
I am not really feeling the name, I think just glass bottle would be good enough.

Reply:
Clashes with parent type, they already share the same unfilled (and when filled with unrecognised reagents) icon, having the same name would be doubly confusing. Its a child type.

Reply:
I get it, but still I don't think "imitation glass bottle" is that good of a name.

Reply:
Fair, open to all to suggest a new name in regular chat or the discord and not here (this bits logged), i got nothing. I really very much would like them to be more distinct but this is the best i can offer at the moment.

Reply:
maybe holobottle?

Reply:
The name has been changed in <https://github.com/The-Merchants-Guild/Merchant-Station-13/pull/222/commits/a81e38a56e472d93e6bbd5c6bf31c633f74c12d7>, holobottle is good enough.